README.md

```diff
@@ -138,6 +138,8 @@ is tagged correctly.
   * `ghcr.io/parkervcp/games:samp`  
 * [`source`](/games/source)
   * `ghcr.io/parkervcp/games:source`
+* [`thebattleofwesnoth`](/games/TheBattleOfWesnoth)
+  * `ghcr.io/parkervcp/games:TheBattleOfWesnoth`
```

Comment:
please use lower case letters here

Reply:
Do I also have to change the whole directory name to have lowercase letters?
![image](https://github.com/parkervcp/yolks/assets/65982675/95ffa732-412f-4f76-88e6-eb19e52a9265)

Reply:
only change it to lowercase

Reply:
It bothered me so I wound up changing every reference to lowercase. I also noticed I misspelled some stuff so it worked out ;).